plugins/generator-1.19.4/forge-1.19.4/templates/overlay.java.ftl

```diff
@@ -82,35 +82,34 @@ package ${package}.client.screens;
             </#if>
 
             <#list data.getComponentsOfType("Image") as component>
-                <#assign x = component.x - 213>
-                <#assign y = component.y - 120>
                 <#if hasProcedure(component.displayCondition)>
                         if (<@procedureOBJToConditionCode component.displayCondition/>) {
                 </#if>
                     RenderSystem.setShaderTexture(0, new ResourceLocation("${modid}:textures/screens/${component.image}"));
-                    Minecraft.getInstance().gui.blit(event.getPoseStack(), posX + ${x}, posY + ${y}, 0, 0,
+					<@calculatePos anchor_point=component.anchorPoint.name() comp_x=component.x comp_y=component.y/>
+                    Minecraft.getInstance().gui.blit(event.getPoseStack(), posX, posY, 0, 0,
                         ${component.getWidth(w.getWorkspace())}, ${component.getHeight(w.getWorkspace())},
                         ${component.getWidth(w.getWorkspace())}, ${component.getHeight(w.getWorkspace())});
                 <#if hasProcedure(component.displayCondition)>}</#if>
             </#list>
 
             <#list data.getComponentsOfType("Label") as component>
-                <#assign x = component.x - 213>
-                <#assign y = component.y - 120>
-                    <#if hasProcedure(component.displayCondition)>
-                        if (<@procedureOBJToConditionCode component.displayCondition/>)
-                    </#if>
-                    Minecraft.getInstance().font.draw(event.getPoseStack(),
-                        <#if hasProcedure(component.text)><@procedureOBJToStringCode component.text/><#else>Component.translatable("gui.${modid}.${registryname}.${component.getName()}")</#if>,
-                        posX + ${x}, posY + ${y}, ${component.color.getRGB()});
+                <#if hasProcedure(component.displayCondition)>
+                    if (<@procedureOBJToConditionCode component.displayCondition/>)
+                </#if>
+				<@calculatePos anchor_point=component.anchorPoint.name() comp_x=component.x comp_y=component.y/>
+                Minecraft.getInstance().font.draw(event.getPoseStack(),

```

Comment:
Spaces are still used here (and on some other lines in both generators)

Reply:
Yea... almost every line is indented with spaces, that's the way the file was before I even touched it.
In my first commit, I used tabs, you didn't like that so i switched to spaces, after i swtiched to spaces you told me to use tabs, so i made sure all the lines i changed something in are using tabs.
I don't know what you want me to do, because what I do, it's wrong..
How about I clean up the indentations of all templates in a single PR?

Reply:
> How about I clean up the indentations of all templates in a single PR?

We cleanup indentations during porting templates to a new version. Now leave it as it is for the current version and try to use the same indentation as used around the place you are doing changes.

Reply:
> In my first commit, I used tabs, you didn't like that so i switched to spaces, 

I always wanted tabs, but I might have explained myself uncorrectly.